WordPress/src/main/java/org/wordpress/android/ui/posts/services/PostUploadNotifier.java

```diff
@@ -96,8 +96,10 @@ public void updateNotificationSuccess(PostModel post, SiteModel site, boolean is
 
         // Notification builder
         NotificationCompat.Builder notificationBuilder = new NotificationCompat.Builder(mContext.getApplicationContext());
-        String notificationTitle = (String) (post.isPage() ? mContext.getResources().getText(R.string
-                .page_published) : mContext.getResources().getText(R.string.post_published));
+        String notificationTitle = PostStatus.DRAFT.equals(PostStatus.fromPost(post)) ? (String) mContext.getResources().getText(R.string
```

Comment:
I think the nested ternary operators make this hard to read - could we break this up into a standard `if` (or two)?